tests/arch/gatemate/run-test.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
these tests aren't doing much good, as this script isn't actually *run* from the top-level Makefile, please add it here (and fix the test failures)

Reply:
Tests failed due to the unsupported initilization values for FF cells. I've set the init values in `dfflegalize` and in the simulation models to zero again.

Reply:
hold on here; whatever you do in the flow has to match what the hardware actually supports

does the hardware always initialize registers to 0 after all?

if yes, the techmap rule for dlatch needs to be updated to use the removeinit thing

if no, please go back to using `x` in `dfflegalize` and sim models, then fix the tests to not use features that the hardware cannot support

Reply:
There is no hardware feature to pre-initialize registers during configuration. After the reset registers are zero. It can however happen that the value changes during the configuration of the CPEs. So I'd also rather use `x` in `dfflegalize` and sim models and adjust the tests.

Since it's only the `initial` blocks in `common/dffs.v` and `common/adffs.v` that lead to errors, I would add the `dffs` and `adffs` tests as verilog without `initial` to the gatemate test directory. That ok?

Reply:
if you cannot guarantee 0 initialization, then yes, this is the correct course of action

I'd slightly prefer reusing the same test `.v` files, just with the `initial` statements wrapped in `ifndef`

Reply:
Done as suggested in 19335d2